docs/en/installing/install-deploy.md

```diff
@@ -149,7 +149,7 @@ BROKER does not currently have, nor does it need, priority\ networks. Broker's s
 
 * Start FE
 
-	`sh bin/start_fe.sh --daemon`
+	`bash bin/start_fe.sh --daemon`
```

Comment:
I see that start_fe.sh has '#!/usr/bin/env bash'. So system will use bash call this shell.
So I think following will be better.
```suggestion
	`./bin/start_fe.sh --daemon`
```